pkg/pipeline/transform/kubernetes/kubernetes.go

```diff
@@ -189,7 +244,7 @@ func (k KubeData) InitFromConfig(kubeConfigPath string) error {
 	return nil
 }
 
-func (k KubeData) initInformers(client kubernetes.Interface) error {
+func (k *KubeData) initInformers(client kubernetes.Interface) error {
 	//defer close(stopChan)
```

Comment:
Is this line no longer relevant? Delete.

Reply:
removed, done

Reply:
done